splink/linker.py

```diff
@@ -1442,7 +1461,7 @@ def _self_link(self) -> SplinkDataFrame:
         self._self_link_mode = True
 
         # Block on uid i.e. create pairwise record comparisons where the uid matches
-        uid_cols = self._settings_obj._unique_id_input_columns
+        uid_cols = self._settings_obj._unique_id_8s
```

Comment:
presuming this is an accidental change - seems to be why the tests are failing

Reply:
I must have fat fingered while I was updating it yesterday.